lib/cash-out/cash-out-atomic.js

```diff
@@ -53,6 +53,7 @@ function preProcess (t, oldTx, newTx, pi) {
         return cashOutActions.logAction(t, 'provisionAddress', rec, addressedTx)
       })
       .catch(err => {
+        // TODO This should send a error notification
```

Comment:
Should I send a notification on a provisionAddress error?

Reply:
Yes, I think so.